magma/inputgenerator.py

```diff
@@ -272,6 +272,10 @@ def _match_argument_file(self, arg_obj, run_obj):
                 #end if
             #end for
             gather = getattr(arg_obj, 'gather', 0)
+            # If not gather look for gather_input
+            if not gather:
+                gather += getattr(arg_obj, 'gather_input', 0)
+            #end if
             gather += getattr(arg_obj, 'extra_dimension', 0)
```

Comment:
Do we want this to work with `gather_input` like this? If so, best to include in a test.

Reply:
they are two different concepts and they could work together, ``extra_dimension`` should apply on top of ``gather`` the same way as it would do for ``gather_input``

Reply:
Probably best to capture in a test; should be a quick addition.